### PR TITLE
Removing [dashboard] and [slice] titles to show name

### PR DESCRIPTION
**Pull request by michellethomas**
Showing [dashboard] and [slice] hide the name of the dashboard or slice in the tab. Anyone object to removing these labels?

@john-bodley @graceguo-supercat @mistercrunch

### CODE REVIEW

**Comment by timifasubaa** [comment]
LGTM

File: superset/views/core.py [hunk 1]
```diff
@@ -1297,7 +1297,7 @@ def explore(self, datasource_type=None, datasource_id=None):
             if datasource_type == 'table' \
             else datasource.datasource_name
         if slc:
-            title = '[slice] ' + slc.slice_name
+            title = slc.slice_name
         else:
             title = '[explore] ' + table_name
```
Comment (williaster) [inline superset/views/core.py]: do you think we should change this as well?

**Comment by williaster** [comment]
thank god 👏 LGTM with one comment on possibly also removing on explore.

**Comment by elibrumbaugh** [comment]
Good call @williaster !

Thank you for making this happen @michellethomas !!!

**Comment by michellethomas** [comment]
@williaster I added that originally but it just shows the table name if you are in explore which might just look a little confusing. I could also add [explore] at the end like `table_name [explore]`

**Comment by williaster** [comment]
what about just "Superset" in that case? Or `Superset - ${tableName}`?

**Comment by michellethomas** [comment]
@williaster All of the crud pages list Superset as the title so there'd be no distinguishing explore from crud pages. What about just `explore - table_name`. I think just table_name could be ok too.

**Comment by williaster** [comment]
oh that makes sense. Yeah I think `Explore - ${tableName}` could work (++ uppercasing it) or just the table name 👍

**Pull request merged**